src/Components/Recommendations/BookRecCards.js

```diff
@@ -0,0 +1,90 @@
+import PropTypes from 'prop-types';
```

Comment:
Rather than using an additional component here, it would be best to reuse the book card and carousels from other places! Can you delete this extra component?